samples/client/petstore-security-test/csharp/SwaggerClient/src/IO.Swagger/Client/JsonSubTypes.cs

```diff
@@ -0,0 +1,281 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace JsonSubTypes
+{
+    //  Copyright 2017 Emmanuel Counasse
```

Comment:
@manuc66 What about putting down something similar to https://github.com/swagger-api/swagger-codegen/blob/64df75d8b6bcf3c2320e822671a0b1ed7d2efa1c/modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java#L2931 ?